src/Reflection/SignatureMap/functionMap.php

```diff
@@ -11223,16 +11223,17 @@
 'ssdeep_fuzzy_hash_filename' => ['string', 'file_name'=>'string'],
 'ssh2_auth_agent' => ['bool', 'session'=>'resource', 'username'=>'string'],
 'ssh2_auth_hostbased_file' => ['bool', 'session'=>'resource', 'username'=>'string', 'hostname'=>'string', 'pubkeyfile'=>'string', 'privkeyfile'=>'string', 'passphrase='=>'string', 'local_username='=>'string'],
-'ssh2_auth_none' => ['mixed', 'session'=>'resource', 'username'=>'string'],
+'ssh2_auth_none' => ['true|array', 'session'=>'resource', 'username'=>'string'],
```

Comment:
or maybe `true|string[]`?

Reply:
This is something we could improve in the future: instead of `array`, we could provide better array notations, e.g. `string[]`. When possible, of course

Reply:
changed